react.js

```diff
@@ -1,24 +1,18 @@
 module.exports = {
   extends: ['./index.js', 'prettier/react'],
-  plugins: ['react', 'sort-imports-es6-autofix', 'react-hooks'],
   parserOptions: {
     ecmaFeatures: {
       jsx: true,
     },
   },
-  settings: {
-    react: {
-      version: 'detect',
-    },
-  },
+  plugins: ['react', 'sort-imports-es6-autofix', 'react-hooks'],
   rules: {
-    'react-hooks/exhaustive-deps': 'warn',
-    'react-hooks/rules-of-hooks': 'error',
     'react/jsx-boolean-value': 'error',
     'react/jsx-key': 'error',
     'react/jsx-no-duplicate-props': 'error',
     'react/jsx-no-undef': 'error',
-    'react/jsx-sort-props': 'off',
+    'react/jsx-sort-default-props': 'error',
+    'react/jsx-sort-props': 'error',
```

Comment:
See also db7a16daa30716984e781de08a36d88fea2d3945.  Only pointing out that there are different opinions on this one.  I don't have any strongly held opinions on these things as long as they are auto-fixable.

Reply:
I just think the more rules we have the less the codebases can differ and the easier it is to switch between them going forward in Brian's and Quinn's idea to have teams less siloed in distinct projects 🤷

Reply:
I also understand there are different opinions on this one, at first I kind of disliked the sorting (as overkill), but now that I have gotten used to it it helps to find stuff more easily and read code better